docker/test/integration/resources/mqtt_broker/mosquitto_test.conf

```diff
@@ -0,0 +1,874 @@
+# Config file for mosquitto
```

Comment:
There seems to be only 3 lines not commented out in this configuration. Could we only leave those in this test config? I think it would be more clear at glance what is actually set.

Reply:
This config is a slightly modified version of the base image, eclipse-mosquitto:2.0.11. Maybe I could apply some small diff on that instead in the Dockerfile in a RUN section.

Reply:
That should be okay as well, or a `sed` command if that's a very trivial change.